src/test/java/org/entando/entando/plugins/jpredis/RedisTestUtils.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2022-Present Entando Inc. (http://www.entando.com) All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+package org.entando.entando.plugins.jpredis;
+
+import com.agiletec.aps.system.common.notify.NotifyManager;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+
+/**
+ * @author E.Santoboni
+ */
+public class RedisTestUtils {
+
+	public static void startContainer(boolean sentinel) throws Exception {
+        ProcessBuilder processBuilder = new ProcessBuilder();
+        if (sentinel) {
+            processBuilder.command("bash", "-c", "docker compose --file docker-compose-sentinel.yaml up --scale redis-sentinel=3 -d ");
+        } else {
+            processBuilder.command("bash", "-c", "docker compose --file docker-compose-single.yaml up -d ");
+        }
+        executeCommand(processBuilder);
+	}
+    
+    public static void stopContainer() throws Exception {
+        ProcessBuilder processBuilder = new ProcessBuilder();
+        processBuilder.command("bash", "-c", "docker-compose stop && docker-compose rm -f");

```

Comment:
I would use `docker compose` also here, as it is the new default instead of `docker-compose`.

Reply:
Done